app/qml/dialogs/MMMonthlyContributorsLimitDialog.qml

```diff
@@ -0,0 +1,38 @@
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+import QtQuick
+
+import "../components" as MMComponents
+
+
+MMComponents.MMDrawerDialog {
+  id: root
+
+  required property bool apiSupportsSubscription
+
+  signal manageAccountClicked()
+
+  title: qsTr( "You've reached the maximum number of active monthly contributors for your current subscription." )
+  imageSource: __style.reachedMonthlyContributorLimitImage
+
+  description: qsTr( "Upgrade your subscription or wait until next month for the limit to reset." )
+
+  primaryButton.text: apiSupportsSubscription ? qsTr( "Manage account" ) : qsTr( "Upgrade" )
```

Comment:
Other texts here also do not make sense when the API does not support subscription, so let's leave it out
```suggestion
  primaryButton.text: qsTr( "Upgrade" )
```

Reply:
Removed the need for support subscription and link to pricing page